test/TestComponents.re

```diff
@@ -0,0 +1,90 @@
+module Box = {
```

Comment:
Maybe we should colocate these components with the main components. Maybe Inside `module Test` in a given component module? We could do `Div.Test`.